ui/src/routes/workspaces.$workspaceId_.projects_.$projectId.lazy.tsx

```diff
@@ -109,3 +113,60 @@ const EditorComponent = () => {
     />
   );
 };
+
+const ErrorComponent = ({ onRefresh }: { onRefresh: () => void }) => {
+  const [accessToken, setAccessToken] = useState<string | undefined>(undefined);
+  const [openVersionDialog, setOpenVersionDialog] = useState(false);
+  const { getAccessToken } = useAuth();
+  const t = useT();
+  useEffect(() => {
+    if (!accessToken) {
+      (async () => {
+        const token = await getAccessToken();
+        setAccessToken(token);
+      })();
+    }
+  }, [accessToken, getAccessToken]);
```

Comment:
This + the accessToken local state doesn't seem necessary. Outside of this useEffect you don't seem to be using the accessToken anyways ❓